src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProject.cs

```diff
@@ -168,8 +180,8 @@ private IScriptExecutor ScriptExecutor
                 _project.AddOrUpdateLegacyCSProjPackage(
                     packageId,
                     range.MinVersion.ToNormalizedString(),
```

Comment:
the MinVersion on the range may be null and this will throw. Example: ``( , 9.0.1]``

I suggest adding tests around this